project-koku/masu
# Database accessor context manager

**Overview**
Switching database accessors to operate as a context manager.  At the exit of the `with` blocks, the sessions will be closed.  The higher layer users of the database accessors no longer have to close db sessions.


**Testing**
1.  Add OCP and AWS providers
2.  Add OCP rates
3. Ingest with masu and verify all is successful and connections are cleaned up

**Test Results**
Closes #315
